src/routes/tools/tools.json

```diff
@@ -322,5 +322,14 @@
 		"addedOn": "2021-12-08",
 		"tags": [],
 		"stars": 0
-	}
+	},
+	{
+		"title": "walk-and-graph-svelte-components",
+		"url": "https://github.com/j2l/walk-and-graph-svelte-components",
+		"description": "CLI node script to walk svelte and js files, to draw a beautiful JPG of your dependencies aka \"imports\". No external dependency, only FS (Node). ",
+		"addedOn": "2022-02-03",
+		"category": "Debugging",
+		"tags": [],
+		"stars": 0
+}
```

Comment:
This line is not well formatted.

You can run `npm run format` to fix it automatically.
And `npm run lint` yo check if everything is ok

Reply:
Which one?
BTW, it's generated by your tool so I leave you fixing it, thanks.